src/baldr/tz_alt.cpp

```diff
@@ -114,6 +114,7 @@
 #include <cctype>
 #include <cstdlib>
 #include <cstring>
+#include <cwchar>
```

Comment:
long story here. i know there was a thread about this when i did it originally but instead of taking the implementation inside of the date library directly. we copy it and modify it to not need to use CURL/filesystem to grab timezone database information but instead we package that data as strings with the build (as we do many things). this allows us to ship the library with all of the data it needs to run (other than the actual tiles).

anyway i updated the submodule to the latest version and in doing so had to redo my hacks for loading tzdb from strings.

Reply:
the other thing this does is it ensure every platform has the same idea about what timezones exist where. faster, per-platform  implementations exist but they dont agree that ET is index 109 for example which is a problem for us because we bake the timezone index into the tile data. so by making the data static we ensure everyone agrees on timezones.